docs/targets/awss3.md

```diff
@@ -59,10 +62,27 @@ The response [CloudEvent][ce] would have the following payload:
 
 | Name | Value | Description |
 |---|---|---|
-|**ce-type**|io.triggermesh.targets.aws.s3.result|Denotes a response payload from S3|
-|**ce-source**|`arn:aws:s3:...`|The S3 ARN value as configured by the target|
-|**body**|[JSON][ce-jsonformat]|A JSON response from the Target invocation|
+|**ce-type**|io.triggermesh.targets.aws.s3.result|Denotes a response payload from the S3 put request|
+|**ce-source**|`arn:aws:s3:...`|The S3's bucket ARN value as configured by the target|
+|**body**|[JSON][ce-jsonformat]|A JSON response from the Target invocation with the Etag associated with the request|
+
+### * Types
```

Comment:
I feel super confused reading this. I'd just start the "Event types" paragraph with a clear description of the behavior such as:

> This event target accepts **any type of event**. The CloudEvent `subject` determines ... (describe as you did). If the event context doesn't contain a subject, ... (describe how you did).
>
> One special case are events of type `...s3.put`, which can contain any payload just like other events, but always cause the CloudEvent context to be discarded, regardless of the value of the `Discard CE attributes` parameter.

And that's it, it wouldn't add anything else. I think it's short and to the point, there is no complicated spec to decipher because not relevant for this target.

wdyt?

Reply:
Upon rereading this, I would agree that it is kinda muddled. especially trying to force a generic type and specific type definition. Hopefully this update will clean things up.